pkg/api/v1/pod/util.go

```diff
@@ -257,9 +257,18 @@ func GetPodCondition(status *v1.PodStatus, conditionType v1.PodConditionType) (i
 	if status == nil {
 		return -1, nil
 	}
-	for i := range status.Conditions {
-		if status.Conditions[i].Type == conditionType {
-			return i, &status.Conditions[i]
+	return GetPodConditionFromList(status.Conditions, conditionType)
+}
+
+// GetPodConditionFromList extracts the provided condition from the given list of condition and returns that.
+// Returns nil and -1 if the condition is not present, and the index of the located condition.
```

Comment:
s/Returns nil and -1/Returns -1 and nil
Remove `, and the index of the located condition.`